about.md

```diff
@@ -6,59 +6,31 @@ title: About
 ## What is the ESMValTool?
 
 ![ESMValTool-logo](/assets/img/EVA-logo.png){:style="float: left;margin-right: 16px;margin-top: 16px;"}
-The Earth System Model Evaluation Tool (ESMValTool) is a community
-diagnostics and performance metrics tool for the evaluation of Earth
-System Models (ESMs) that allows for routine comparison of single or
-multiple models, either against predecessor versions or against
-observations. The priority of the effort so far has been to target
-specific scientific themes focusing on selected Essential Climate
-Variables, a range of known systematic biases common to ESMs, such as
-coupled tropical climate variability, monsoons, Southern Ocean
-processes, continental dry biases and soil hydrology-climate
-interactions, as well as atmospheric CO2 budgets, tropospheric and
-stratospheric ozone, and tropospheric aerosols. The tool is being
-developed in such a way that additional analyses can easily be added.
-A set of standard recipes for each scientific topic reproduces specific
-sets of diagnostics or performance metrics that have demonstrated their
-importance in ESM evaluation in the peer-reviewed literature. The
-ESMValTool is a community effort open to both users and developers
-encouraging open exchange of diagnostic source code and evaluation
-results from the CMIP ensemble. This will facilitate and improve ESM
-evaluation beyond the state-of-the-art and aims at supporting such
-activities within the Coupled Model Intercomparison Project (CMIP) and
-at individual modeling centers. Ultimately, we envisage running the
-ESMValTool alongside the Earth System Grid Federation (ESGF) as part of
-a more routine evaluation of CMIP model simulations while utilizing
-observations available in standard formats (obs4MIPs) or provided by
-the user.
+
+The Earth System Model Evaluation Tool (ESMValTool) is an open-source community-developed diagnostics
+and performance metrics tool for the evaluation and analysis of Earth System Models (ESMs). ESMValTool
+allows for a comparison of single or multiple models, either against predecessor versions or against
```

Comment:
Can we say `against both predecessor versions and observations` because it is not quite an either or comparison?

Reply:
Good point! Changed as suggested.